sound/soc/fsl/Kconfig

```diff
@@ -75,6 +75,14 @@ config SND_SOC_FSL_ESAI
 	  This option is only useful for out-of-tree drivers since
 	  in-tree drivers select it automatically.
 
+config SND_SOC_FSL_DAI
+	tristate "Generic FSL DAI support for Sound Open Firmware"
```

Comment:
shouldn't this be selected when you enable SOF on i.MX? If you expose this as an option, I can guarantee buildbots will find a config there you have missed dependencies.

Reply:
@plbossart indeed I should add a select SND_SOC_FSL_DAI under SND_SOC_SOF_IMX8_SUPPORT. Although, to be honest I'm really scared on touching Kconfig dependencies.

So, is this what you have in mind? I think this makes sense.

```
diff --git a/sound/soc/sof/imx/Kconfig b/sound/soc/sof/imx/Kconfig
index bae4f7bf5f75..b8a73194c869 100644
--- a/sound/soc/sof/imx/Kconfig
+++ b/sound/soc/sof/imx/Kconfig
@@ -15,6 +15,7 @@ config SND_SOC_SOF_IMX8_SUPPORT
        bool "SOF support for i.MX8"
        depends on IMX_SCU
        depends on IMX_DSP
+       select SND_SOC_FSL_DAI
        help
          This adds support for Sound Open F
```

Reply:
yep